sycl/unittests/config/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Enable exception handling for these unit tests
+set(LLVM_REQUIRES_EH 1)
```

Comment:
Could you please clarify why a separate test suite is being created for the config tests?

Reply:
Sorry, can't understand, can you please elaborate?

Reply:
The patch creates a new tests suite - executable. Why not adding this test case to the already existing one, e.g. `sycl/unittests/misc`?

Reply:
Does it dramatically affect the build time of the check-sycl? I don't think so.

Reply:
> How much is it important to avoid creation of separate gtest binaries?
I think it's important. At least to understand the reason why it is in a separate binary.

The situation when we create a separate test binary for each test case doesn't seem like a correct usage of google test framework.

Reply:
I can keep the "config" folder (to not create a dump in the "misk" folder) and move the content of config/CMakeLists.txt to sycl/unittests/misc CMakeLists.txt. What do you think about it? 
Also, can I update sycl/unittests/CMakeLists.txt for all unit tests so that we create only one binar for all tests and the folder structure remained the same?

Reply:
> I can keep the "config" folder (to not create a dump in the "misk" folder) and move the content of config/CMakeLists.txt to sycl/unittests/misc CMakeLists.txt. What do you think about it?

What if the issue with creating "a dump" in the misc directory?

> Also, can I update sycl/unittests/CMakeLists.txt for all unit tests so that we create only one binar for all tests and the folder structure remained the same?

Please, do not do that.

Reply:
> What if the issue with creating "a dump" in the misc directory?

It's the matter of readability inside of unittests folder: one can take a look in it and find the config folder which means that we have unittests for SYCL config. If the source code of the test is located in the mics directory, it wouldn't be so clear. Moreover, this folder can contain lots of other tests in the future, and the idea is to get rid of it - to create some order inside unittests folder. 

What @denis-kabanov suggests is placing SYCL config unittests sources in unittests/config folder, and (if we need less test binaries) compile these sources with misc tests by including SYCL config unittests source file to mics's CMakeLists.

If readability and some order in unittests folder doesn't matter, @denis-kabanov can put SYCL config unittests sources to mics folder.

Reply:
Okey, I'll leave the test untouched in the "config" folder.